gc-cli/bin/gc-cli.js

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env node
 "use strict";
 
+require("babel-register");
```

Comment:
I think you need to add `ignore: false` like in this code here https://github.com/caplin/brjs-app-converter/blob/master/src/cli.js#L7 otherwise if people install the tool without using it via the repo it won't work.